x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/openai/OpenAiService.java

```diff
@@ -165,7 +165,7 @@ private OpenAiEmbeddingsModel updateModelWithEmbeddingDetails(OpenAiEmbeddingsMo
             model.getServiceSettings().organizationId(),
             SimilarityMeasure.DOT_PRODUCT,
             embeddingSize,
-            null
+            model.getServiceSettings().maxInputTokens()
```

Comment:
Double check me, we do want to grab this from the original model right?

Reply:
👍 yes I missed that one